scripts/deploy/lib/deployment-utils.js

```diff
@@ -14,6 +14,10 @@ const buildDatacenterUploadPath = (datacenter, packageName, version, extension =
 // ex: datadog-rum.js
 const buildBundleFileName = (packageName, extension = 'js') => `datadog-${packageName}.${extension}`
 
+// ex: pull-request/2781/datadog-rum.js
+function buildPullRequestUploadPath(packageName, version, extension = 'js') {
+  return `PullRequest/${version}/datadog-${packageName}.${extension}`
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏if `PullRequest` is use in an URL, it could be nice to use `pull-request` instead for readability sake.
It is also what is advertised in the comment.